<launch>
    <include file="$(find-pkg-share realsense2_camera)/launch/rs_launch.py">
        <arg name="align_depth.enable" value="true"/>
        <!-- <arg name="depth_module.depth_profile" value="640x480x60"/>
        <arg name="rgb_camera.color_profile" value="640x480x60"/> -->
    </include>

    <node pkg="vlm_vision" exec="frame_recorder">
        <param name="output_dir" value="/tmp/pipeline_eval"/>
        <param name="max_frames" value="50"/>
    </node>
</launch>